cli/src/command.rs

```diff
@@ -252,7 +252,15 @@ pub fn run() -> Result<()> {
 			})
 		},
 		Some(Subcommand::ValidationWorker(cmd)) => {
-			let _ = sc_cli::init_logger("", sc_tracing::TracingReceiver::Log, None, false);
+			let _ = sc_cli::init_logger(
+				sc_cli::InitLoggerParams {
+					pattern: "".into(),
+					tracing_receiver: Default::default(),
+					tracing_targets: None,
+					disable_log_reloading: false,
+					disable_log_color: true,
```

Comment:
I'm not sure why disabling color here by default, but I don't really know this validationworker command.